posts/_markdown/quarc.md

```diff
@@ -0,0 +1,24 @@
+---
+title: QUARC
+date: January 23, 2023
+author: Christoph Kirsch and Stefanie Muroya Lei
+---
+
+Bounded Model Checking (BMC) is a well-established technique used to ensure the correctness of critical software. Given a number *n* representing the number of state transitions, it builds the Finite State Machine (FSM) of a program and searches for the input(s) that make a set *S* of machine states (e.g. division by zero, non-zero exit codes) reachable within *n* state transitions.
```

Comment:
Please consider making the first paragraph even more general with respect to the introduction. 

Examples: Is the tool you are building a compiler, a debugger, an optimizer?

A possible suggestion:
"Bounded Model Checking (BMC) is a well-established technique used to ensure the correctness of critical software. **Some examples are...** **Our aim with in this project is...**"